charts/kafka-operator/templates/crds.yaml

```diff
@@ -978,7 +978,9 @@ spec:
                         type: string
                       description: 'Custom labels for the broker pods, example use
                         case: for Prometheus monitoring to capture the group for each
-                        broker as a label, e.g.: kafka_broker_group: "default_group"'
+                        broker as a label, e.g.: kafka_broker_group: "default_group"
+                        these labels won''t override the reserved labels that the
```

Comment:
```suggestion
                        these labels won't override the reserved labels that the
```

Reply:
@Kuvesz You will need to update this line: https://github.com/banzaicloud/koperator/blob/master/api/v1beta1/kafkacluster_types.go#L174, e.g. `won't` -> `will not`, and then run `make manifests`. 

It seems `controller-gen` does not like contracted forms and I forgot to run `make manifests` when I added this line